### ISSUE TITLE
Status and Updates too wordy

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-policies-prototype#12]
1. "Last updated" could be a single line, no need for a full sentence. Actually, we might have "last updated" for every entry as it is ambiguous on the page level.
2. We should have a button/link to an input form for updates, like we have on the [tools list](https://www.w3.org/WAI/ER/tools/). Maybe also here have an additional link for every item that would pre-fill the form for easier updating

**Comment by bakkenb** [comment]
Agree with Shadi on the first point here. If I see that there was a new 'last updated' date since I last visited, I would have no idea of <b>what content</b> was last updated. It would be nice to know if a specific country or state/province was updated, and would be more relevant to what information I am looking for.

Maybe the paragraph could work better this way:

> This page is updated on a regular basis as new information is received. You will see a "last updated" date for each country to inform you of how current the data is. Know of new or changed laws or policies not listed, or corrections needed, [submit updates to Web Accessibility Policies(link)].

**Comment by slhenry** [comment]
Agree to delete "This page was updated on 03 Feb 2017 and will be updated on a regular basis as new information is received."

See also https://github.com/w3c/wai-policies-prototype/issues/7

**Comment by yatil** [comment]
You can make edits to the texts here: https://github.com/w3c/wai-policies-prototype/tree/gh-pages/_help

**Comment by yatil** [comment]
@nitedog I’ll put your point number 2 into a separate issue so it is easier to follow.

**Comment by dboudreau** [comment]
+1 for the shortest, most straightforward wording possible.

**Comment by maryjom** [comment]
To clarify, this is about the Status and Updates section at the top of the page since update information will be throughout the page. Remove that section, and simply have a button/link to submit updates somewhere toward the top of the page.

**Comment by slhenry** [comment]
Discussion in [10 Feb telecon](https://www.w3.org/2017/02/10-eo-minutes.html#item08)

**Comment by maryjom** [comment]
The status and updates part has been edited to shorten it significantly.  The headings are going to be collapsed into one heading with the pertinent information for the page which is being tracked with issue #13 and the addition of a working submission button is being tracked with issue #18.  I'll copy over comments about potentially having a submit updates button on individual country parts to pre-fill the submission form into Issue #18 to keep these issues more distinct.
